Firestore/CHANGELOG.md

```diff
@@ -2,9 +2,9 @@
 
 # v0.16.1
 - [fixed] Offline persistence now properly records schema downgrades. This is a
-  forward-looking change that allows versions past this one to safely downgrade
-  to this version. Downgrading to versions prior this one can be safe depending
-  on the source version. For example, downgrading from v0.16.1 to v0.15.0 is
+  forward-looking change that allows all subsequent versions to safely downgrade
+  to this version. Depending on the source version, you can also downgrade to
```

Comment:
I'm torn about this change. The language flows better, which is an improvement. However, "you can also downgrade" does not convey the inherent danger I'm trying to convey. 

Basically downgrades between some source and target versions could be safe, but it's not like this change enables that. I worry that this language is too permissive sounding.